themes/powerline/powerline.theme.bash

```diff
@@ -37,13 +37,17 @@ function set_rgb_color {
 }
 
 function powerline_shell_prompt {
+    SHELL_PROMPT_COLOR=${SHELL_THEME_PROMPT_COLOR}
+    if sudo -n uptime 2>&1 |grep -q "load"; then
```

Comment:
This seems to work better - thanks for changing this.

One nit-picking comment: Could you please add a space character between the pipe and the `grep` command?

It would also be great if you could make the same change in the _powerline-multiline_ theme, so both use the same check.

Thanks!